src/dotnet-compile-fsc/Program.cs

```diff
@@ -85,6 +85,7 @@ public static int Main(string[] args)
             bool targetNetCore = 
                 commonOptions.Defines.Contains("DNXCORE50") ||
                 commonOptions.Defines.Where(d => d.StartsWith("NETSTANDARDAPP1_")).Any() ||
```

Comment:
Yeah it probably should, but I'll let the FSC guys take care of that, they have already begun taking this to a new repo.

FYI @enricosada